### ISSUE TITLE
`mc admin user list --json` output doesn't include user groups

### ISSUE THREAD

**Issue by zhurlik** [minio/mc#4433]
## Expected behavior
It would be great to have the same user details json for both commands:
- `mc admin user list --json`
- `mc admin user info --json`

## Actual behavior
The field `memberOf` is missing from `mc admin user list` for each user

## Steps to reproduce the behavior
1. sh-4.4# `mc admin user info --json test-minio user-927`
```
{
 "status": "success",
 "accessKey": "user-927",
 "policyName": "readonly,consoleAdmin",
 "userStatus": "enabled",
 "memberOf": [
  "admin"
 ]
}
```
2. sh-4.4# `mc admin user list --json test-minio | grep 927`
```{"status":"success","accessKey":"user-927","policyName":"readonly,consoleAdmin","userStatus":"enabled"}```

## mc --version
- (paste output of `mc --version`)
```
mc version RELEASE.2022-11-17T21-20-39Z (commit-id=2770f13a73c5e98ae1a0665c76cf099f5f578a13)
Runtime: go1.19.3 linux/amd64
Copyright (c) 2015-2022 MinIO, Inc.
License GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
```
## System information
```
Ubuntu 22.04, 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
docker images
REPOSITORY            TAG       IMAGE ID       CREATED       SIZE
quay.io/minio/minio   latest    60628873a9f4   5 weeks ago   294MB
minio/mc              latest    d511a44ed0ed   6 weeks ago   198MB
```

**Comment by kannappanr** [comment]
@zhurlik Will check and get back to you

**Comment by harshavardhana** [comment]
Fixed via #4463
